setup.py

```diff
@@ -129,9 +132,14 @@ def py2exe_options():
     author_email=author_email,
     classifiers=filter(None, classifiers.split('\n')),
     license=license,
-    scripts=['openerp-server', 'odoo.py'],
+    scripts=['openerp-server'],
+    entry_points={
+        'console_scripts': [
+            'odoo-bin=odoo.cli.odoobin:main',
```

Comment:
@odony any option is good for me. 

I actually ended up doing a few commits on this topic because I needed to update setup.py and get odoo.py out of the way because it conflicts with the odoo package.

Actually this is somewhat unrelated with the title of this PR and is best discussed on #13413